src/storm/builder/ExplicitModelBuilder.cpp

```diff
@@ -41,10 +41,18 @@ namespace storm {
         template<typename StateType>
         StateType ExplicitStateLookup<StateType>::lookup(std::map<storm::expressions::Variable, storm::expressions::Expression> const& stateDescription) const {
             auto cs = storm::generator::createCompressedState(this->varInfo, stateDescription, true);
-            STORM_LOG_THROW(stateToId.contains(cs), storm::exceptions::IllegalArgumentException, "State unknown.");
+            //TODO search once
```

Comment:
No, I do not want to add it. I just default to return the size.  Optimally, the StateToId would allow me to get it or return a signal that it did not find this. However, this may not be an issue due to code optimization. Given that the explicit-state-lookup is more a debugging facility, I don't care too much about performance.